src/Game.ts

```diff
@@ -271,9 +256,11 @@ export class Game {
 
 				this.checkGuess(currSelection as number);
 
-				this.findOperatives().filter(op => op.team === sop.team).forEach(
-					innerOp => this.toggleCard(innerOp, currSelection)
-				)
+				gu.getByTeam(gu.getOperatives(this.agents), sop.team).forEach(op => 
+						this.toggleCard(op, currSelection))
```

Comment:
This filter should be closed by a semicolon, and could be moved to one line. I'd also suggest adding an inline comment explaining what this does, as I'm not sure.